### ISSUE TITLE
Dataset to data package

### ISSUE THREAD

**Issue by maier-m** [NCEAS/metacatui#632]
From Arctic Team meeting May 25, 2018 https://docs.google.com/document/d/1WbB_GRCo-hivgBintHxr-aMwEpecMZhu-YSDngiLRtM/edit#heading=h.venbqe88n71o... the term data package was determined to be more appropriate than dataset or data set when talking about a collection of data and metadata.

Accordingly, the inconsistent `Data Set` and `dataset` seen below should be changed to `Data Package` and `data packages` to have consistent messaging.

<img width="1001" alt="screen shot 2018-05-25 at 1 36 16 pm" src="https://user-images.githubusercontent.com/16710796/40565268-ab67b3c4-6020-11e8-9cfb-57b25f2497f1.png">

**Comment by maier-m** [comment]
Also occurs for obsoleted packages "A newer version of this dataset exists"

**Comment by amoeba** [comment]
Good catch. I'm not sure what the preferred nomenclature would be here. What do others think?

**Comment by maier-m** [comment]
From the Arctic Team meeting May 25, 2018, the group coalesced on "data package" (although not everyone was able to attend that meeting). This is more important as we update data submission guidelines just to make sure we are consistent across all platforms.

**Comment by maier-m** [comment]
It was a small bullet under "Submission guidelines review". I don't recall capitalization discussed. Maybe it needs to be.
